docs/components/ResponsiveContainer.tsx

```diff
@@ -45,7 +45,7 @@ export const ResponsiveContainer = ({ children }: { children?: ReactNode }) => {
           value={containerHeight[0]}
           max={maxUnits}
           min={10}
-          onChange={(nextValue) => setHeight([nextValue] as number[])}
+          onChange={(_event, nextValue) => setHeight([nextValue as number])}
```

Comment:
Is this type cast needed?

Reply:
can be string or number, it's a lazy change as it's just a storybook dev thing?